tests/Custom/SelectTest.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Behat\Mink\Tests\Driver\Custom;
```

Comment:
No problem for me, move this test to Mink repo, however I assume that you understand that other drivers would be affected.